deliverables/README.md

```diff
@@ -1,14 +1,19 @@
 ## Deliverables
 #### Required
 ##### Questions
-- What is a framework?
-- What's the difference between a programming language and a framework?
-- What’s the difference between a library and a framework?
+- **What is a framework?**  
+- Is an abstraction in which software providing generic functionality can be selectively changed by additional user-written code, providing application-specific software.
+- 
+- **What's the difference between a programming language and a framework?**
+- A framework or an engine is something someone else has already programmed for you that you can use to do your work easier.
```

Comment:
No a framework is like the foundation -- its glue that holds everything together

a library is reusable collection of methods/functions that makes development faster, easier, better